docs/en/integrations/index.mdx

```diff
@@ -41,7 +41,7 @@ Core integrations are built or maintained by ClickHouse. These integrations are
 |Go|<GoLangSVG style={{width: '3rem'}} />|Language client|The Go client uses the native interface for a performant, low-overhead means of connecting to ClickHouse.|[Documentation](https://github.com/clickhouse/clickHouse-go#readme)|
 |Java|<JavaSVG style={{width: '3rem', 'background-color': 'white'}} />|Language client|The Java client is an async, lightweight, and low-overhead library for ClickHouse.|[Documentation](https://github.com/clickhouse/clickhouse-jdbc#readme)|
 |Kafka|<KafkaSVG style={{width: '3rem'}} />|Data ingestion|Bidirectional integration with Apache Kafka, the open-source distributed event streaming platform.|[Documentation](./kafka)|
-|Node.JS|<NodeSVG style={{width: '3rem', 'background-color': 'white', 'height': '3rem'}} />|Data ingestion|Data streaming Node.js Streaming API.|[Documentation](https://www.npmjs.com/package/@clickhouse/client)|
```

Comment:
@DanRoscigno, this change might be required for other logos: java, superset, DBeaver, Spark, Python:
![2022-09-26_12-50-55](https://user-images.githubusercontent.com/3198181/192258759-987e8fae-c50a-43fe-aa92-60fa0727ccd7.png)
![2022-09-26_12-51-31](https://user-images.githubusercontent.com/3198181/192258836-915ab04d-d41b-4b90-96f5-e1b87a916618.png)

Reply:
Do you have images that look good in dark and light modes? @mshustov